### PR TITLE
Customise viewport size

### PR DESCRIPTION
**Pull request by camillobruni**

### CODE REVIEW

**Comment by camillobruni** [comment]
* Increase the viewport size from 800px by 600px to 1500px by 800px.
* Use the rendered size of the viewport when calculating whether the window size is too small

**Review by flashdesignory — APPROVED** [review]
LGTM 👍  - it's challenging not to hard-code values in too many places and to be consistent.

**Comment by bgrins** [comment]
I agree with this in spirit, though would like to check with @Bas-moz that this is going to work in our CI environment. Correct me if I'm wrong, but did we decide to switch to 1366x768 to tighten it up a little bit and match a common desktop resolution?

**Comment by camillobruni** [comment]
PTAL again

File: tests/benchmark-runner-tests.mjs [hunk 1]
```diff
@@ -83,7 +83,9 @@ describe("BenchmarkRunner", () => {
         });
 
         describe("_appendFrame", () => {
-            it("should create an absolutely positioned iframe with 800px x 600px dimensions", async () => {
+            const DEFAULT_WIDTH = 1366;
```
Comment (flashdesignory) [inline tests/benchmark-runner-tests.mjs:86]: I like the refactor a lot!!
Could we re-use the params class and add a failsafe for searchParams if needed? 
Alternatively we could export the hardcoded values from that file and have it all live in the same place? 🤷

File: resources/params.mjs [hunk 2]
```diff
@@ -0,0 +1,42 @@
+
+class Params {
+    viewport = {
+        width: 1366,
```
Comment (rniwa) [inline resources/params.mjs:4]: Can we still with 800x600 for now?

Comment (bgrins) [reply resources/params.mjs:4]: +1 to decoupling the size change into another issue/PR, then we can get the refactor landed now

File: tests/benchmark-runner-tests.mjs (hunk 1 above)
Comment (camillobruni) [reply tests/benchmark-runner-tests.mjs]: DONE: Added a `defaultParams` export on the params.mjs module.

**Review by bgrins — APPROVED** [review]

**Review by rniwa — APPROVED** [review]

**Pull request merged**